Apps/Database/src/Delegates/DBFeedbackDelegate.cs

```diff
@@ -114,10 +114,13 @@ public DBResult<List<UserFeedbackAdmin>> GetAllUserFeedbackEntries()
 
             foreach (UserFeedbackAdmin feedbackAdmin in feedback)
             {
-                string? email;
-                if (profileEmails.TryGetValue(feedbackAdmin.UserProfileId, out email))
+                if (!string.IsNullOrWhiteSpace(feedbackAdmin.UserProfileId))
                 {
-                    feedbackAdmin.Email = email != null ? email : string.Empty;
```

Comment:
= email ?? String.Empty;

Reply:
agree, there are other improvements on the qery too that can be done. For now, let's keep it simple to only address the exception